lib/core/index.tsx

```diff
@@ -0,0 +1,14 @@
+// Copyright 2023 @paritytech/polkadot-dashboard-ui authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { StyledComponentInterface } from "../types";
+
+export const Entry = ({
+  children,
+  style,
+  className,
+}: StyledComponentInterface) => (
+  <div className={"core-entry " + className} style={style}>
```

Comment:
can you use `valEmpty` util to ensure that an empty string is in place when `className` is not defined.